generators/utils.js

```diff
@@ -18,6 +18,8 @@
  */
 const _ = require('lodash');
 const toPascalCase = require('to-pascal-case');
+const buildEnumInfo = require('generator-jhipster/generators/utils').buildEnumInfo;
```

Comment:
buildEnumInfo is depracated can you use getEnumInfo ?

Reply:
I just changed. Still waiting for CI checks to complete